source/crud/read.txt

```diff
@@ -0,0 +1,166 @@
+.. _mongosh-read:
+
+===============
+Query Documents
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+Use the :method:`db.collection.find()` method in the |mdb-shell|
+to query documents in a collection.
+
+.. include:: /includes/fact-sample-data-examples.rst
+
+.. TODO: Make a callout here to the Connect to an Atlas Cluster section
+   in the connection docs
+
+Select All Documents in a Collection
+------------------------------------
+
+To select all documents in the collection, pass an empty document as the
+query filter parameter to the find method. The query filter parameter
+determines the select criteria.
+
```

Comment:
This seems to return only 20 documents if the collection has more than 20. Behavior seems different from the legacy shell where you can type `it` to view additional documents. Might be worth noting.

Reply:
cc @mmarcon, this PR is already merged but I forgot to ask about this. When a read operation returns more than 20 docs in the mongodb shell, are we expected to be able to iterate over those docs with a cursor? if not, what is the expected behavior?